budgetportal/tests/test_bulk_upload.py

```diff
@@ -15,12 +15,6 @@
 EMAIL = "testuser@domain.com"
 PASSWORD = "12345"
 
-# Hacky make sure we don't call out to openspending.
```

Comment:
Hmmm the "hacky" came to bite us - I think we should find a better solution to this that doesn't break the tests you're busy writing, but still doesn't call out to other apps -this expands scope for you so we should discuss when and how we add that.

I think continue as is and I should just make sure to add to the scope and explain how we should handle this.